### ISSUE TITLE
Error Code 1095 and execute it many times, before it succeeds

### ISSUE THREAD

**Issue by Citrullin** [iota-community/c-iota-workshop#3]
```bazel run -c opt examples:send_hello```
needs to execute it a couple of times, before it succeeds. Randomly works.

**Comment by andrea-tesei** [comment]
@Citrullin same problem here. Have you solved this issue somehow?

**Comment by Citrullin** [comment]
@andrea-tesei Not yet, but the library also changed. So, upgrading it to the new version might help. Is it important for your work?

**Comment by andrea-tesei** [comment]
@Citrullin thank you for your answer. I've updated the library to the latest commit but still I've the same problem. 

I've also tried to deploy a Private Tangle, and use this code to attach an hello_world transaction. As you said in your first comment, sometimes the transactions succeed. But in the majority of the cases I'm receiving an "ERROR API:285 - API Validation failed: Invalid trytes input" error message. 

If I will find out a solution I will post it here. Any suggestions will be appreciated. Thanks

**Comment by Citrullin** [comment]
@andrea-tesei We moved the C library from entangled to [iota.c](https://github.com/iotaledger/iota.c)
So, this makes a lot of things different. So, I guess, I need to move forward with porting it to the new library then.

**Comment by andrea-tesei** [comment]
Yeah I see. But still I've succeed on publishing transactions only with entangled version, and with specific commit of the example contained in this repository (wtf?) and ONLY with my Private Tangle instance. 

I've tried to use new C library (iota.c) but I always receive the error above. I'm doing some work for research purposes and I want to try C library to see if I can reach better performances in terms of transaction publication delay. 

Btw, I'm continue to use entangled version to complete my experiments (now stucked in Trytes manipulation -.-), but if you move forward with the porting or if you have other examples/hint/suggestions I will appreaciate if you notify me. Thanks

**Comment by Citrullin** [comment]
@andrea-tesei There will probably no Ternary in the new version of IOTA anymore. Coordicide is currently without ternary and I guess they will continue without it. So, you don't need to care too much about ternary anymore. Just use the helper functions for now.

**Comment by bernardoaraujor** [comment]
`send_tokens` is also broken.

`receive_hello` works sometimes, fails sometimes.
